docs/man/auth/client-credentials.md

```diff
@@ -23,14 +23,20 @@ in the OS keyring for future use.
 
 ## Examples
 
+Authenticate with client credentials (id and secret provided interactively)
+
+```shell
+opentdf auth client-credentials
```

Comment:
Is there a reason for renaming the CLI here from `otdfctl` to `opentdf`?

Reply:
Yeah not sure how or why I did that but I reverted them back to `otdfctl`